src/components/ProjetoCard.tsx

```
@@ -1,21 +1,23 @@
 import { Projeto } from '@/commons/interfaces'
 import { Card } from '@/components/ui/card'
 import { Button } from '@/components/ui/button'
-import { Users, ClipboardList, Clock } from 'lucide-react'
+import { FaUsers } from "react-icons/fa";
+import { MdGroupAdd } from "react-icons/md";
 
 interface ProjetoCardProps {
   projeto: Projeto
-  horas?: number
+  vagasDisponiveis?: number
+  defaultImage?: string;
```

ESLint

[error] Delete `;`

(prettier/prettier)